atc/exec/put_step_test.go

```diff
@@ -61,7 +61,7 @@ var _ = Describe("PutStep", func() {
 		repo  *build.Repository
 		state *execfakes.FakeRunState
 
-		putStep *exec.PutStep
+		putStep exec.Step
```

Comment:
Assuming this was also unintentional like the other PutStep change.

Reply:
Same.